classes/ObjectModel.php

```diff
@@ -1872,19 +1872,32 @@ public function deleteImage($force_delete = false)
      * Checks if an object exists in database.
      *
      * @param int $id_entity
-     * @param string $table
+     * @param string $table Deprecated since 1.7.8.x
```

Comment:
Could use `Tools::displayParameterAsDeprecated` if $table is not null.

Reply:
Done ! Thanks